src/main/java/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;
+
+import dao.ProductDao;
+import dao.impl.ProductDaoImpl;
+import model.Product;
+import service.ProductService;
+
+import java.util.List;
+
+public class ProductServiceImpl implements ProductService {
+    private static final ProductDao productDao = new ProductDaoImpl();
```

Comment:
we need to use factory here
and use this class in controllers
the same about user

Reply:
done